pkg/identities/handlers.go

```diff
@@ -45,8 +45,9 @@ func (a *API) RegisterEndpoints(mux *chi.Mux) {
 	mux.Put("/api/v0/identities/{id:.+}", a.handleUpdate)
 	// mux.Patch("/api/v0/identities/{id:.+}", a.handlePartialUpdate)
 	mux.Delete("/api/v0/identities/{id:.+}", a.handleRemove)
+	mux.Delete("/api/v0/identities/sessions/{id:.+}", a.handleSessionDisable)
```

Comment:
A minor question: Is this API for compensation?

Reply:
Question: we want to disable kratos session **on logout** right? So the logic to do that (which is now in "handleSessionDisable") should be part of the logout process, not a new API endpoint. That's what I was expecting, but I may be missing pieces

Reply:
```suggestion
	mux.Delete("/api/v0/identities/{id:.+}sessions/{sid:..+}", a.handleSessionDisable)
	mux.Delete("/api/v0/identities/{id:.+}sessions", a.handleSessionDisableAll)
```

for these endpoints you need to add related authorization permissions
ideally we need the caller to have access to the user with `can_view` and `can_edit`